src/contacts/contacts.py

```diff
@@ -71,11 +71,13 @@ def get_contacts(username):
                 raise PermissionError
 
             contacts_list = contacts_db.get_contacts(username)
+            app.logger.debug("Succesfully retrieved contacts.")
             return jsonify(contacts_list), 200
         except (PermissionError, jwt.exceptions.InvalidTokenError):
+            app.logger.error("Error retrieving contacts list: authentication denied")
```

Comment:
I would actually print the `InvalidTokenError` here like you're doing with the `SQLAlchemyError` below -- and just make sure nothing specific about the incoming JWT is logged to stdout